src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,27 @@
+package lotto.view;
+
+import lotto.util.NumberUtil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static int inputAmount() {
+        System.out.println("구입금액을 입력해 주세요.");
+        return NumberUtil.getPositiveInteger(SCANNER.nextLine());
+    }
+
+    public static List<Integer> inputWinnigNumber() {
+        System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+        String inputText = SCANNER.nextLine();
+        String[] tokens = inputText.split(",");
+
+        return Arrays.stream(tokens).map(String::trim).map(NumberUtil::getPositiveInteger).collect(Collectors.toList());
```

Comment:
스트림 문법은 개행을 하면 가독성에 도움이 되는 것 같아요 😃 
```suggestion
        return Arrays.stream(tokens)
                .map(String::trim)
                .map(NumberUtil::getPositiveInteger)
                .collect(Collectors.toList());
```

Reply:
아래 커밋에 반영했습니다. 
https://github.com/next-step/java-lotto/pull/2673/commits/1fac73ef91ac0e3c62479b4bc4b0fb72f45fe6ec